src/detail-page/components/LearnList.tsx

```diff
@@ -0,0 +1,73 @@
+import Button from '@material-ui/core/Button';
+import React from 'react';
+import styled from 'styled-components';
+import { ProfessionKey } from '../../data/home-page/ITProfessions';
+import { getData } from '../hooks/useLearnList';
+
+interface LearnListProps {
+    professionId: ProfessionKey;
+  }
+
+const StyledDiv = styled('div')`
+    display: flex;
+    flex-direction: column;
+    justify-content: space-between;
+    align-items: flex-start;
+`;
+
+const StyledButton = styled(Button)`
+    border-radius: 0;
+    font-weight: bold;
+    text-transform: none;
+    border: 2px solid ${props => props.theme.palette.primary.main};
+
+    &:hover {
+        border: 2px solid ${props => props.theme.palette.primary.main};
+    }
+`;
+
+const StyledList = styled('ul')`
+    list-style: none;
+    margin-top: 0;
+    margin-bottom: 1rem;
+    padding-left: 1.5rem;
+`;
+
+const StyledLi = styled('li')`
+    margin: .8rem 0;
+    font-size: .95rem;
+    position: relative;
+
+    &::first-letter {
+        text-transform: uppercase;
+    }
+
+    &::before {
+        content: '';
+        width: .4rem;
+        height: .4rem;
+        border-radius: 50%;
+        background-color: ${props => props.theme.palette.primary.main};
+        position: absolute;
+        left: -1.5rem;
+        transform: translateY(6px)
+    }
+`;
+
+export const LearnList: React.FC<LearnListProps> = ({ professionId }) => {
+  const data = getData(professionId);
+  return (
+    <StyledDiv>
+      <StyledList>
+        {data.list.map((item, index) => (
```

Comment:
We are not using index, so we don't have to declare it as param :)

Reply:
Yes, I want to use it as a key, but our linter doesn't want to cooperate so i use item instead, but maybe I will fight with it, and change key for index, I think it will be a little bit better in this case.

Reply:
Done :)